src/components/ChatWidget.svelte

```
@@ -78,15 +79,17 @@
       }
```

Lines 72-80: If `savedMessages is corrupted, the widget silently falls back to empty state. The user loses their conversation history without any indication.

Consider showing an error toast or clearing the corrupted key explicitly.